explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -340,26 +346,28 @@ object ObsTabTiles:
               .mapN: (site, asterism, duration) =>
                 props.posAngleConstraint match
                   case PosAngleConstraint.AverageParallactic =>
-                    // See also `anglesToTestAt` in AladinCell.scala.
-                    averageParallacticAngle(
+                    val avpa = averageParallacticAngle(
```

Comment:
Ah. So for `AverageParallacticAngle`, the flip of the angle is allowed. Good detective work.

Reply:
yeah it was all well documented of course